uwdata/lyra
# Remove MOST unused variables

eslint identified a number of required modules or instantiated variables that were never used after being created. Many of these looked to be the result of copy-pasted code or of modules that had their purpose refined over time: d3 was left over in a component that didn't use it, or datalib was required as a development aid but then omitted from the final module code.

These are easy to clean up and the app appears to run and build without them; I did leave a few instances of this error in place, however, because there are spots where the variable assignment may be generating side effects or signaling a logical error elsewhere and those deserve to be surfaced through more thorough testing.

Steps to test:
- Build Lyra and test basic functionality to see whether I broke anything!

LYRA-261
